### PR TITLE
chore(#8306): remove the last of grunt

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

[description]

medic/cht-core#8306

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8306-remove-grunt-7/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8306-remove-grunt-7/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8306-remove-grunt-7/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by garethbowen** [comment]
@dianabarsan You're probably too busy but wanted to give you a chance to review if you had time! Let me know and I can volunteer someone else.

**Review by dianabarsan — APPROVED** [review]
I won't stand in the way of progress here :D 
I know grunt was dead and needed to go, but omg the state of npm scripts, if anything is spagetti code, it's this :D

File: package.json
Comment (garethbowen) [reply package.json:36]: From my POV it's the same just horizontal instead of vertical. I agree that it still sucks.

File: scripts/build/index.js
Comment (garethbowen) [reply scripts/build/index.js:217]: TIL! Thanks.

File: package.json
Comment (garethbowen) [reply package.json:39]: Would it be better to pull out every one of the "sub scripts" and make it a bash script, so this file just contained the "entry points" for dev and ci and nothing else?

Comment (dianabarsan) [reply package.json:39]: It's possible. I really think that horizontal scroll is the worst. 
Is it a lot of work to try?

Comment (garethbowen) [reply package.json:39]: Not really. I'll have a look next week, possible after merging this...

**Comment by latin-panda** [comment]
@garethbowen dev cmds are working fine in my mac os, I tried building on top of existing and also from scratch. All unit test cmds are working fine as well.

**Comment by garethbowen** [comment]
Thanks @latin-panda ! I'll merge it next week when I have a chance to communicate it to everyone.

**Pull request merged**
